src/vs/workbench/services/issue/electron-sandbox/issueService.ts

```diff
@@ -82,6 +88,37 @@ export class NativeIssueService implements IWorkbenchIssueService {
 			const result = [this._providers.has(extensionId.toLowerCase()), this._handlers.has(extensionId.toLowerCase())];
 			ipcRenderer.send('vscode:triggerReporterStatusResponse', result);
 		});
+		ipcRenderer.on('vscode:triggerReporterMenu', async (event, arg) => {
+			if (!this.isRunning) {
```

Comment:
isn't this just `this.extensionIdentifierSet.size() !== 0`

Reply:
just remove `isRunning` entirely. will not run actions when requested again